pyannote/audio/applications/multilabel.py

```diff
@@ -0,0 +1,584 @@
+        return app
+
+    def validate_init(self, protocol_name, subset='development'):
+        if self.label in self.task_.labels_spec["regular"]:
+            derivation_type = "regular"
+        elif self.label in self.task_.labels_spec["union"]:
+            derivation_type = "union"
+        elif self.label in self.task_.labels_spec["intersection"]:
+            derivation_type = "intersection"
+        else:
+            raise ValueError("%s not found in training labels : %s"
+                             % (self.label, self.task_.label_names))
+
+        protocol = get_protocol(protocol_name, progress=False,
+                                preprocessors=self.preprocessors_)
+        files = getattr(protocol, subset)()
+
+        self.pool_ = mp.Pool(mp.cpu_count())
+
+        # pre-compute features for each validation files
+        validation_data = []
+        for current_file in tqdm(files, desc='Feature extraction'):
+            # precompute features
+            if not isinstance(self.feature_extraction_, Precomputed): 
+                current_file['features'] = self.feature_extraction_(
+                    current_file)
+
+            if derivation_type == "regular":
+                current_file[self.label+"_ref"] = current_file["annotation"].subset([self.label])
+            else:
+                current_file[self.label+"_ref"] = MultilabelTask.derives_label(current_file["annotation"],
+                                                                derivation_type=derivation_type,
+                                                                meta_label=self.label,
+                                                                regular_labels=self.task_.labels_spec[derivation_type][self.label])
+            validation_data.append(current_file)
+        return validation_data
+
+    def validate_epoch(self, epoch, protocol_name, subset='development', validation_data=None):
+        """
+        Validate function given a class which must belongs to ["CHI", "FEM", "KCHI", "MAL", "SPEECH", "OVERLAP"]
+        """
+        # Name of the class that needs to be validated
+        class_name = self.label
+
+        # load model for current epoch
+        model = self.load_model(epoch).to(self.device)
+        model.eval()
+
+        # compute (and store) SAD scores
+        duration = self.task_.duration
+
+        sequence_labeling = SequenceLabeling(
+            model=model, feature_extraction=self.feature_extraction_,
+            duration=duration, step=.25 * duration, batch_size=self.batch_size,
+            device=self.device)
+
+        for current_file in validation_data:
+            scores = sequence_labeling(current_file)
+
+            # We extract the score of interest
+            dimension = self.task_.label_names.index(class_name)
+            scores_data = scores.data[:, dimension].reshape(-1, 1)
+
+            current_file[class_name+'_scores'] = SlidingWindowFeature(
```

Comment:
It is my understanding that there is only one `label` validated at a time. 
Why do we need to add it as a prefix to `'scores'`?